cocos/gfx/gl-constants.ts

```diff
@@ -0,0 +1,562 @@
+/* eslint-disable @typescript-eslint/no-duplicate-enum-values */
+export enum WebGLConstants {
```

Comment:
What is this `gl-constants.ts` file for?

Reply:
Shall we use `const enum`, since these values will never be changed.

Reply:
Currently, both enum and `const enum` will be `inlined` since I wrote a custom babel plugin in ccbuild.
I remember that I got same errors when using `const enum`.

The default WebGL constants is not enum which will not be inlined. So I add this file to inline the enums.